### PR TITLE
Fixing issue #627

### PR DESCRIPTION
**Pull request by melibleq**
Adds format gVCF to `EDAM_dev.owl`.

### CODE REVIEW

**Review by matuskalas — CHANGES_REQUESTED** [review]
Thank you sooo much for your super quick PR @melibleq 🚀 and sorry for my super slow review 🙇🏽‍♂️ You really got the EDAM concepts editing right 🙌🏽 The TODOs to finish this up are:

- [ ] Could you please create a new feature branch for this (from `main`) and move the changes to `EDAM_dev.owl` there? If not by copy&pasting, you can do it the Git way by: new branch from `main` + cherry-pick the 2 commits.
- [x] Please make sure to add `Co-authored-by: Steve Laurie <steven.laurie@cnag.crg.eu>` after an empty line(!) to the bottom of the next commit(s) you'll do here. (Either commit without the `-m` option, or insert a second `-m "NEWLINE_WITHOUT_CLOSING_DOUBLE-QUOTES` and fill it in there.)
- [ ] 2 XML "complex elements" are missing their closing tags. Can you see them? The Travis-CI test does fail there, can you find that in Travis?
- [x] The `created_in` needs to be 1.26.

MILLION THANKS!!! 🙏🏽

A great way to test your addition visually is to upload your `EDAM_dev.owl` file as a new ontology to WebProtégé, see if it loads without errors, and check how your new concept looks there (_e.g._ are the super-concepts right, are the related concept right, ... ?). You can also include a screenshot to the PR, or share your test ontology on WebProtégé and include a link to the PR. The latter options are useful especially for bigger additions or changes.

**Comment by melibleq** [comment]
> Thank you sooo much for your super quick PR @melibleq  and sorry for my super slow review 🙇🏽‍♂️ You really got the EDAM concepts editing right 🙌🏽 


Haha no problem at all 😁 so glad I got the idea correctly!

> * [ ]  Could you please create a new feature branch for this (from `main`) and move the changes to `EDAM_dev.owl` there? If not by copy&pasting, you can do it the Git way by: new branch from `main` + cherry-pick the 2 commits.
> * [ ]  Please make sure to add `Co-authored-by: Steve Laurie <steven.laurie@cnag.crg.eu>` after an empty line(!) to the bottom of the next commit(s) you'll do here. (Either commit without the `-m` option, or insert a second `-m "NEWLINE_WITHOUT_CLOSING_DOUBLE-QUOTES` and fill it in there.)
> * [ ]  2 XML "complex elements" are missing their closing tags. Can you see them? The Travis-CI test does fail there, can you find that in Travis?
> * [ ]  The `created_in` needs to be 1.26.

I'll work on these today! So we're working on release 1.26 and that's why that should be the version? I used the currently stable one (1.25).

> MILLION THANKS!!! 🙏🏽
> 
> A great way to test your addition visually is to upload your `EDAM_dev.owl` file as a new ontology to WebProtégé, see if it loads without errors, and check how your new concept looks there (_e.g._ are the super-concepts right, are the related concept right, ... ?). You can also include a screenshot to the PR, or share your test ontology on WebProtégé and include a link to the PR. The latter options are useful especially for bigger additions or changes.

Thank you for this kind and detailed review!!! I'll test it and let you know what happened 😺

File: EDAM_dev.owl [hunk 1]
```diff
@@ -37269,6 +37269,33 @@ experiments employing a combination of technologies.</oboInOwl:hasDefinition>
...
 
+    <!-- http://edamontology.org/format_XXXX -->
+
+    <owl:Class rdf:about="http://edamontology.org/format_XXXX">
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/format_2921"/>
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/format_3016"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://edamontology.org/is_format_of"/>
+                <owl:someValuesFrom rdf:resource="http://edamontology.org/data_3498"/>
```
Comment (matuskalas) [inline EDAM_dev.owl:37280]: Special appreciation for adding the _is_format_of_! 🚀🙌🏽😸

Comment (melibleq) [reply EDAM_dev.owl:37280]: :grin: I made sure to read the [documentation](https://edamontologydocs.readthedocs.io/en/latest/developers_guide.html#mandatory-attributes) about attributes, especially formats

**Comment by matuskalas** [comment]
I marked what has been fixed in b3f1ee9.

**Comment by matuskalas** [comment]
@melibleq's commits were cherry-picked and merged in #695, so I'm closing this PR now.

**Pull request closed without merging**